### DISCUSSION TITLE
Is it possible to retrieve the atoms that set R/S or E/Z stereo tags?

### DISCUSSION THREAD

**Discussion by eguidotti** [rdkit/rdkit#6452] [Q&A] [answered]
I’m wondering whether the RDKit allows to retrieve the atoms that set stereo tags (R/S for atoms and E/Z for bonds). For instance, if atom 1 is R, is there some function that retrieves the neighbors that have been used to disambiguate the stereochemistry and return their indices?

**Comment by ptosco** [comment]
@eguidotti For bonds you may use
[rdkit.Chem.rdchem.Bond.GetStereoAtoms](http://rdkit.org/docs/source/rdkit.Chem.rdchem.html?highlight=bond#rdkit.Chem.rdchem.Bond.GetStereoAtoms)
For atoms you may retrieve the neighbours with [rdkit.Chem.rdchem.Atom.GetNeighbors](http://rdkit.org/docs/source/rdkit.Chem.rdchem.html?highlight=getneighbors#rdkit.Chem.rdchem.Atom.GetNeighbors)

**Reply by eguidotti** [reply]
Thanks @ptosco As far as I understand, those functions return the immediate neighbors. I’m wondering whether there is a function to retrieve all neighbors that are used to disambiguate the stereo tag. For instance, the function should return also “neighbors of neighbors” when atoms that are directly attached to a stereo center are constitutionally identical (because we need longer chains of substituents to assign priority and resolve the stereo tag)

**Comment by greglandrum** [comment, marked as answer]
@eguidotti this is not possible in the RDKit
